addons/website_event_sale/tests/test_website_event_sale_pricelist.py

```diff
@@ -67,3 +68,34 @@ def test_pricelist_different_currency(self):
             self.WebsiteSaleController.pricelist(promo=None)
             self.so._cart_update(line_id=so_line.id, product_id=self.ticket.product_id.id, set_qty=1)
         self.assertEqual(so_line.price_reduce_taxexcl, 900, 'Incorrect amount based on the pricelist and its currency.')
+
+class TestWebsiteEventSaleorder(HttpCase, TestWebsiteEventSaleCommon):
```

Comment:
A bit misplaced inside "test_website_event_sale_pricelist.py" as it does not have much to do with pricelist.

You could have a "test_website_event_sale.py" file, a "TestWebsiteEventSale" class and a "test_website_event_sale" method.
As you're testing the main extension of the flow here.